lib/helpers.js

```diff
@@ -4,7 +4,9 @@
 
 var debug = require("debug")("poynt");
 var Jwt = require("jsonwebtoken");
-var makeRequest = require("request");
+var makeRequest = require('retry-request', {
```

Comment:
use double quotes

Reply:
also can you have this only used if a flag is passed into the initialization of this library? it's dangerous to auto retry all requests as especially transaction-related ones can result in double/multiple charges for the customer. ideally auto retry is opt-in only, and also always ignored for certain calls (specifically https://github.com/poynt/poynt-node/blob/master/lib/collect.js https://github.com/poynt/poynt-node/blob/master/lib/transactions.js)

Reply:
I heard some requests should persist the poynt-request-id does that need to be accounted for?